.github/dependabot.yml

```diff
@@ -0,0 +1,19 @@
+version: 2
+updates:
+  - package-ecosystem: "npm"
+    directory: "/"
+    schedule:
+      interval: "daily"
```

Comment:
Daily feels a little aggressive, but we can give it a try..

Reply:
Is it? The upside is that updates are more spread out, but this could also be annoying.